spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformanceAttributes.scala

```diff
@@ -48,6 +49,11 @@ trait HyperConformanceAttributes extends HasComponentAttributes {
       PropertyMetadata("Menas API URL retry count",
         Some("How many times a call to Menas API URL should be retried after failure before proceeding to the next URL. E.g. 2"),
         required = false),
+    menasOptionallyRetryableExceptions ->
+      PropertyMetadata("Menas API optionally retryable HTTP exceptions",
+        Some("""List of HTTP status codes that are allowed to be retryable, e.g. [404,402] (retry count is """ +
```

Comment:
Would list all supported ones, so the user doesn't need to investigate which ones are allowed.